index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <meta property="og:title" content="Linkbrary" key="title" />
+    <meta
+      name="description"
+      property="og:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요"
+      key="description"
+    />
+    <meta
+      property="og:image"
+      content="./images/library.PNG"
+      key="image"
+    />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@linkbrary" />
+    <meta name="twitter:title" content="Linkbrary" />
+    <meta name="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta
+      name="twitter:image"
+      content="./images/library.PNG"
+    />
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+      <nav> 
+        <div class="toolbar">
+          <a href="index.html">
+            <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+          </a>
+          <a class="login" href="signin.html">
+            <span>로그인</span>
+          </a>
+        </div>
+      </nav> 
+
+
+        <div class="bg">
+            <div class="title">
+                <span class="grad">세상의 모든 정보</span><span class="black">를<br>쉽게 저장하고 관리해 보세요</span>
+            </div>
+
+            <div class="addlink" onclick="location.href='/signup.html'">
+                링크 추가하기
+            </div>
+
+            <div class="img1">
+                <img class="img1_" src="./images/img1.png">
+            </div>
+
+        </div>
+
+        <div class="bg2">
+            <div class="frame">
+                <div class="text1">
+                    <p><span class="grad2">원하는 링크</span><span class="black2">를 저장하세요</span>
+                    <p><span class="small1">나중에 읽고 싶은 글, 다시 보고 싶은 영상, 사고 싶은 옷, 기억하고 싶은 모든 것을 한 공간에 저장하세요.</span>
```

Comment:
유지보수 및 가독성을 위한 코드의 일관성 측면에서 닫는 `p` 태그를 다는게 좋을 것 같아요. 또, h1과 마찬가지로 제목 요소처럼 보입니다. h1이 이미 존재하기 때문에 다음인 h2가 적당해 보이네요.

```suggestion
                    <h2><span class="grad2">원하는 링크</span><span class="black2">를 저장하세요</span></h2>
                    <h2><span class="small1">나중에 읽고 싶은 글, 다시 보고 싶은 영상, 사고 싶은 옷, 기억하고 싶은 모든 것을 한 공간에 저장하세요.</span></h2>
```